dangerfile.ts

```diff
@@ -198,7 +198,7 @@ function checkForLicense({ packageJson }: TinaPackage) {
  *
  */
 function fileNeedsLicense(filepath: string) {
-  return new RegExp(/\.(js|tsx?)$/).test(filepath)
+  return new RegExp(/^(?!examples\/).+\.(js|tsx?)$/).test(filepath)
```

Comment:
likely not that relevant since our source files should be typescript, but might as well include `jsx` files

```suggestion
  return new RegExp(/^(?!examples\/).+\.(jsx?|tsx?)$/).test(filepath)
```